### PR TITLE
Removed some vars from window scope

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by danfruehauf** [comment]
Did you verify it works as expected?

**Comment by pmbohm** [comment]
@danfruehauf 
the home/ redirect still works.
the 'drilldownFilterIndex' only occurs in the 'hasDrilldown' function in the whole project.

File: grails-app/views/home/index.gsp [hunk 1]
```diff
@@ -28,11 +28,14 @@ The AODN/IMOS Portal is distributed under the terms of the GNU General Public Li
...
-                window.location.replace(homeWithoutTrailingSlash);
-            }
+
+            (function () {
+                // Fixes #1723, redirect 'home/' to 'home'
+                if (window.location.href.match(/\/home\/$/)) {
+                    var homeWithoutTrailingSlash = window.location.href.substring(0, window.location.href.length - 1);
+                    window.location.replace(homeWithoutTrailingSlash);
+                }
+            }());
```
Comment (dnahodil) [inline grails-app/views/home/index.gsp]: @pmbohm Why this change?

**Comment by pmbohm** [comment]
oh but chrome is not as happy with the home/ redirect!!!

File: grails-app/views/home/index.gsp (hunk 1 above)
Comment (pmbohm) [inline grails-app/views/home/index.gsp]: I was looking at fire bug and i saw a bunch of stuff that I thought was unnecessary to be in the global scope. 
'homeWithoutTrailingSlash' may be necessary!

Comment (dnahodil) [inline grails-app/views/home/index.gsp]: Okay, so it was to change the scope of the variable, @pmbohm?
